### PR TITLE
chore: add msgpack example

### PR DESCRIPTION
**Pull request by kemingy**
Signed-off-by: Keming <kemingy94@gmail.com>

### CODE REVIEW

File: examples/resnet50_server_msgpack.py [hunk 1]
```diff
@@ -38,29 +40,31 @@
...
 
-        img = cv2.resize(img, (256, 256))
-        crop_img = (
-            img[16 : 16 + 224, 16 : 16 + 224].astype(np.float32) / 255
-        )  # center crop
-        crop_img -= [0.485, 0.456, 0.406]
-        crop_img /= [0.229, 0.224, 0.225]
-        crop_img = np.transpose(crop_img, (2, 0, 1))
-        return crop_img
+        tensor = transforms.ToTensor()(image)
```
Comment (lkevinzc) [inline examples/resnet50_server_msgpack.py:65]: Is `ToTensor()` jittable?

Comment (kemingy) [reply examples/resnet50_server_msgpack.py:65]: No. Have to call this one before the jit script.

Comment (lkevinzc) [reply examples/resnet50_server_msgpack.py:65]: Got it. BTW just curious, does the jitted version show significant speed improvement? I have not used PyTorch jit before.

Comment (kemingy) [reply examples/resnet50_server_msgpack.py:65]: Good question. I haven't tested it. 🤣

Comment (lkevinzc) [reply examples/resnet50_server_msgpack.py:65]: Lol, but this example looks fine! In Jax what I know is that jitting can bring much speedup. See their example results here: https://jax.readthedocs.io/en/latest/jax-101/02-jitting.html#jit-compiling-a-function

**Review by lkevinzc — APPROVED** [review]

**Pull request merged**